swift/Arrow/Sources/Arrow/ArrowWriter.swift

```diff
@@ -38,7 +38,7 @@ public class ArrowWriter {
             self.data.append(data)
         }
     }
-
+    
```

Comment:
Are this trailing spaces change and other similar changes intentional?

Reply:
I wasn't too concerned with these style changes as I was going to follow up with a swiftlint PR.  Since we are planning on running swiftlint on the code as a prior PR to this one, then these will be resolved (should be) when swiftlint is run against the files.